azext_edge/edge/providers/check/lnm.py

```diff
@@ -302,13 +314,24 @@ def _process_lnm_pods(
     conditions: Optional[List[str]] = None,
     namespace: Optional[str] = None,
     detail_level: int = ResourceOutputDetailLevel.summary.value,
+    resource_name: str = None,
 ) -> None:
     def _get_lnm_pods_namespace(pod: V1Pod) -> str:
```

Comment:
Similar comments elsewhere, but this could be a common method to avoid re-implementing it everywhere

Reply:
yep it should be gone once I finish fixing the merge conflicts😋